src/components/table/PokemonsTable.component.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { Table,TableBody,TableCell,TableContainer,TableFooter,TablePagination,TableRow,Paper,TableHead } from '@material-ui/core';
+import './table.component.style.css';
+import TablePaginationActions from "./TablePaginationActions";
+import {pokemonService} from '../../service/PokemonService';
+
+export class PokemonsTable extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            page: 0,
+            rowsPerPage: 5,
+            totalPokemons: 0,
+            pokemons: [],
+            headerCells: [
```

Comment:
I think you can just create a constant for header titles and not set it in state.